src/lib/blueprint-features.ts

```diff
@@ -0,0 +1,109 @@
+import { Blueprint } from 'src/stores/wpcom-api';
+
+export interface UnsupportedFeature {
```

Comment:
We don't use it outside
```suggestion
interface UnsupportedFeature {
```